extensions/data-transfer/portability-data-transfer-microsoft/src/test/java/org/dataportabilityproject/transfer/microsoft/helper/MockJobStore.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright 2018 The Data-Portability Project Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.dataportabilityproject.transfer.microsoft.helper;
+
+import org.dataportabilityproject.spi.cloud.storage.JobStore;
+import org.dataportabilityproject.spi.cloud.types.JobAuthorization;
+import org.dataportabilityproject.spi.cloud.types.LegacyPortabilityJob;
+import org.dataportabilityproject.types.transfer.models.DataModel;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
+
+/**
+ * An implementation for testing.
+ */
+public class MockJobStore implements JobStore {
```

Comment:
Can we eventually use the in memory job store when it's ported?

cc : @rtannenbaum

Reply:
Probably can

Reply:
Maybe we can add a comment or an issue for a mockjobstore testing articfact

Reply:
It's ported! If you want to give it a shot: https://github.com/google/data-portability/blob/dc85c0e060cd3f9ba322e482ad7085ea6d62ba41/extensions/cloud/portability-cloud-local/src/main/java/org/dataportabilityproject/cloud/local/InMemoryKeyValueStore.java

Reply:
OK I'll try that in a subsequent pull request.